win32ss/user/ntuser/callback.h

```diff
@@ -70,6 +70,7 @@ BOOL
 APIENTRY
 co_IntGetCharsetInfo(LCID Locale, PCHARSETINFO pCs);
 
+HANDLE FASTCALL co_IntLoadImage(LPCWSTR,UINT,INT,INT,UINT);
```

Comment:
NOTE: This new callback may be what's needed to replace our UserLoadImage function.